### DISCUSSION TITLE
Issues backingup/importing Teslamate

### DISCUSSION THREAD

**Discussion by modkeyIT** [teslamate-org/teslamate#2695] [Q&A] [answered]
When I originally setup Teslamate I followed this guide thinking I would want to externally host.  

https://www.tslaspot.com/teslamate-setup/

Recently I purchased a Synology NAS with Docker and am trying to move it to that.

I followed the instructions for installing Teslamate on Docker, then followed the instructions for backup and restore

https://docs.teslamate.org/docs/installation/docker/
https://docs.teslamate.org/docs/maintenance/backup_restore/

It restored without any errors but I am getting "No Data" in all of my panels.

**Comment by cwanja** [comment]
Were there any errors when you restored? What is in the database logs?

**Reply by modkeyIT** [reply]
No errors when I followed import instructions. Teslmate logs show:

<html>
<body>


2022-06-09 19:21:11 | stdout | 2022-06-09 19:21:11.932 [error] Postgrex.Protocol (#PID<0.2312.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): connection refused - :econnrefused
-- | -- | --
2022-06-09 19:21:08 | stdout | 2022-06-09 19:21:08.966 [error] Postgrex.Protocol (#PID<0.2313.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): non-existing domain - :nxdomain


</body>
</html>

**Comment by modkeyIT** [comment]
I think my issue is because the guide I followed to initially setup teslamate used a password for the DB user.  What do I change within the import data line to add a password to user teslamate?

docker-compose exec -T database psql -U teslamate -d teslamate < teslamate.bck

**Comment by modkeyIT** [comment, marked as answer]
Ended up changing the database password on the originating teslamate DB and exporting/importing to a fresh install on my NAS.  Im sure there was an easier way to do it, but that did it fo rme.
